.github/workflows/run-archiver.yml

```diff
@@ -4,10 +4,7 @@ name: run-archiver
 on:
   workflow_dispatch:
   schedule:
-    - cron: "21 4 * * 0" # Sundays 4:21 AM UTC
-  push:
-    branches:
-      - "daz/**"
+    - cron: "21 4 1 */3 *" # 4:21 AM UTC, 1st of Jan/Apr/Jul/Oct
```

Comment:
Will confirm with RMI what the schedule should be, records are typically updated towards the end of the month.

Reply:
I think we might want to consider oversampling on the archives -- e.g. if we want to actually have workable quarterly archives, trying to generate new archives each month instead.

This would:

* reduce the chances of making an archive right before the data gets updated, and then having to wait a whole quarter or do a manual archiving run to get the fresh data.
* give us an early heads-up when something is broken, so we don't end up shocked that there's a lot of work to do when we get around to trying to do the quarterly update.
* make the system more robust to transient failures due to network hiccups, etc.
* provide a more granular versioned public record of the evolution of these data sources, which are often updated with little or no public explanation.